### PR TITLE
lang: [es] Revision and Update translated to Spanish(Spain) language

### PR DESCRIPTION
**Pull request by celiomarcos**
Revision and Update translated text to Spanish(Spain) language

### CODE REVIEW

**Comment by datamweb** [comment]
Hello, 
Thanks for the PR,
Please run the following command to fix error **PHPCSFixer**:
```console
composer cs-fix
```

File: src/Language/es/Auth.php
Comment (datamweb) [inline src/Language/es/Auth.php:1]: @treborin Can you review?

Comment (treborin) [reply src/Language/es/Auth.php:1]: Hi @datamweb , how is the best way to do it?
I see many words to coorect

Comment (datamweb) [reply src/Language/es/Auth.php:1]: Hey,
See https://youtu.be/lSnbOtw4izI

Comment (treborin) [reply src/Language/es/Auth.php:1]: Well.... thank you. I know how to send a pull request. Done it on Update Auth.php #694

Comment (datamweb) [reply src/Language/es/Auth.php:1]: @treborin , Thanks, I would prefer if you have any suggestions for changes here.
Please learn how to submit a change proposal through the video.
This makes @celiomarcos  also enjoy his first participation.

**Comment by datamweb** [comment]
@celiomarcos This PR still cannot be merged, please run command `composer cs-fix` to fix error PHPCSFixer.

https://github.com/codeigniter4/shield/actions/runs/4547747831/jobs/8018028707?pr=693

If you need more help please let me know.

**Review by datamweb — APPROVED** [review]
@celiomarcos thanks👌

**Review by kenjis — APPROVED** [review]
👍

**Pull request merged**